docs/Maths.md

```diff
@@ -233,3 +233,82 @@ img = Image.fromarray(np.asarray(canvas.buffer_rgba()))
 
 ...
 ```
+### Using Plotly ###
+
+Before running this example, please install the required dependencies using the command below:
+
+```
+pip install fpdf2 plotly==5.13.1 kaleido numpy
```

Comment:
Thank you for your feedback, I am locking the version because in the [plotly getting started page](https://plotly.com/python/getting-started/) they seem to install it in that manner, it seems like 5.13.1 is the latest version anyways as stated in the [pypi](https://pypi.org/project/plotly/) page so I will be removing the version lock. also I have ran the first script and it executes in 1.4 seconds without any hanging on my machine. My environment is Python 3.11 and windows 10.

Reply:
The second script seems to also require the installation of kaleido as plotly uses it when writing the svg figure. I tried running the second script without kaleido and it raised an error so I am not sure why kaleido would hang in the first script on your machine but run on the second script fine.